docs/content/en/docs/operator-manual/piped/configuring-event-watcher.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Configuring event watcher"
+linkTitle: "Configuring event watcher"
+weight: 6
+description: >
+  This page describes how to configure piped to enable event watcher.
+---
+
+To enable [EventWatcher](/docs/user-guide/event-watcher/), you have to configure your piped at first.
+
+### [require] Grant write permission
+The [SSH key used by Piped](/docs/operator-manual/piped/configuration-reference/#git) must be a key with write-access because Event watcher automates the deployment flow by commitng and pushing to your git repository.
```

Comment:
typo: "committing"

Reply:
nit: because `piped` needs to commit and push to your git repository when any incoming event matches.